pkg/security/security.go

```diff
@@ -403,6 +415,21 @@ func GetOSRootFilePath() string {
 	return ""
 }
 
+// CheckWorkloadCertificate returns true when the workload certificate
+// files are present under the provided paths. Otherwise, return false.
+func CheckWorkloadCertificate(certChainFilePath, keyFilePath, rootCertFilePath string) bool {
+	if _, err := os.Stat(certChainFilePath); err != nil {
+		return false
+	}
+	if _, err := os.Stat(keyFilePath); err != nil {
+		return false
+	}
+	if _, err := os.Stat(rootCertFilePath); err != nil {
+		return false
+	}
+	return true
+}
+
```

Comment:
Hi, which part is responsible for checking the certificate expiration time, is it Istio CA or Spire Agent Server? If the certificate expires and Istio's own SDS secretManager not started, what should we do in this scenario?